geonode_mapstore_client/client/js/utils/AppRoutesUtils.js

```diff
@@ -10,8 +10,7 @@ import { ResourceTypes } from '@js/utils/ResourceUtils';
 
 export const appRouteComponentTypes = {
     VIEWER: 'ViewerRoute',
-    SEARCH: 'SearchRoute',
-    DETAIL: 'DetailRoute',
+    CATALOGUE: 'CatalogueRoute',
     DATASET_UPLOAD: 'UploadDatasetRoute',
     DOCUMENT_UPLOAD: 'UploadDocumentRoute',
     HOME: 'HomeRoute'
```

Comment:
Is the home route still in use? We can remove the route if it's no longer in use